From dca9e587e84991adc7652475b5fd8188c6b5160f Mon Sep 17 00:00:00 2001
From: Cam Hutchison <camh@xdna.net>
Date: Fri, 6 Oct 2017 10:02:40 +1100
Subject: [PATCH] env: ifdef out the default environment.

I don't want my uboot environment polluted by the default one. ifdef it
out. This means an environment must be provided on the sd card or as
part of felbooting on sunxi boards.
---
 include/env_default.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/env_default.h b/include/env_default.h
index b574345af2..708aa413fc 100644
--- a/include/env_default.h
+++ b/include/env_default.h
@@ -22,6 +22,8 @@ static char default_environment[] = {
 #else
 const uchar default_environment[] = {
 #endif
+
+#if 0
 #ifdef	CONFIG_ENV_CALLBACK_LIST_DEFAULT
 	ENV_CALLBACK_VAR "=" CONFIG_ENV_CALLBACK_LIST_DEFAULT "\0"
 #endif
@@ -107,7 +109,9 @@ const uchar default_environment[] = {
 #ifdef	CONFIG_EXTRA_ENV_SETTINGS
 	CONFIG_EXTRA_ENV_SETTINGS
 #endif
+#endif
 	"\0"
+
 #ifdef DEFAULT_ENV_INSTANCE_EMBEDDED
 	}
 #endif
-- 
2.11.0

